src/cvc.mli

```diff
@@ -0,0 +1,33 @@
+module RSA :
+sig
+  module Public :
+  sig
+    type t =
+      { n: Z.t
+      ; e: Z.t
+      }
+    [@@deriving ord,yojson,eq,show]
+  end
+end
+
+module ECDSA :
+sig
+  module Public :
+  sig
+    type t =
+      { modulus : Z.t
+      ; coefficient_a : Z.t
+      ; coefficient_b : Z.t
+      ; base_point_g : Z.t
+      ; base_point_r_order : Z.t
+      ; public_point_y : Z.t
+      ; cofactor_f : Z.t
+      }
+    [@@deriving ord,yojson,eq,show]
+  end
+end
+
+type t =
```

Comment:
I added `decode` functions to `RSA.Public` and `ECDSA.Public` since that's where the types are defined. I'm not sure if we'll ever have *non*-public CVC keys to parse, though. Would it make sense to move the type+decoder functions directly into `RSA.blah` and `ECDSA.blah`?